### PR TITLE
Sheel's location changes

### PR DESCRIPTION
**Pull request by twymer**
Just UI changes, note this branch PR's into locations-as-groups, not master. Commits are (mostly) atomic.

https://docs.google.com/a/dimagi.com/spreadsheets/d/1F-dZ-I31ItuQoClyalUiaS5oFu5WaZiZhtDFMBUOagk

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: corehq/apps/locations/templates/locations/manage/locations.html [hunk 1]
```diff
@@ -115,76 +127,87 @@
...
+{% if has_location_types %}
+    <script type="text/html" id="location-template">
+
+        <div class="loc_section">
+            <!-- header bar -->
+            <div class="loc_header" data-bind="if: name() != '_root'">
+                <div class="btn-group" style="float:right;">
+                    {% if not show_inactive %}
+                        <a class="btn btn-primary" data-bind="attr: { href: loc_edit_url(uuid()) }">
+                            Edit
```
Comment (czue) [inline corehq/apps/locations/templates/locations/manage/locations.html]: trans

File: corehq/apps/locations/templates/locations/manage/locations.html [hunk 2]
```diff
@@ -115,76 +127,87 @@
...
-            {% endif %}
-
         </div>
-    </div>
 
-</script>
+    </script>
+{% else %}
+  <div class="loc_child no_children alert alert-info">
+    Please define the location types
```
Comment (czue) [inline corehq/apps/locations/templates/locations/manage/locations.html]: trans

File: corehq/apps/locations/views.py [hunk 3]
```diff
@@ -493,6 +495,9 @@ def location_export(request, domain):
 @domain_admin_required
 @require_POST
 def sync_facilities(request, domain):
+    # TODO this is believed to be obsolete and should
+    # likely be removed, just need to make sure it isn't
+    # magically used by ils/ews first..
```
Comment (czue) [inline corehq/apps/locations/views.py]: it's not, and we can probably just remove it.

**Comment by czue** [comment]
good to go pending tiny feedback and one confirmation on spec

**Pull request merged**